cmdstanpy/lib.py

```diff
@@ -399,6 +399,10 @@ def __init__(self, run: Dict = None, csv_files: Tuple[str] = None) -> None:
         """sampler output csv files."""
         self._sample = None
         """assembled draws across all chains, stored column major."""
+        self._metric = None
+        """per-chain metrics."""
```

Comment:
This is not really that important, but I have used to read comments above the line with `#` before the comment.

Also, if these are comments for users then they should be put in docstring. There is no limit for verbosity.

Reply:
removing this set of comments - redundant with comments on the corresponding @property accessors.